website/docs/docs/supported-databases/redshift-v2.md

```diff
@@ -13,7 +13,7 @@ In this guide, we'll create credentials for one user, and for our production dep
 * An already operating Amazon Redshift cluster.
 * Raw data loaded into your Redshift cluster
 * A set of superuser privileges (including database connection details)
-* Privileges in AWS that allow you whitelist an IP address
+* Privileges in AWS that allow you allow web traffic from an IP address
```

Comment:
This sentence feels a little clunk

Reply:
ah, that's my bad. We actually don't link to this doc in the sidebar presently... you cool with me just deleting the file? I think this was a hidden doc that got carried over in the Readme migration

Reply:
sure!